astrowidgets/core.py

```diff
@@ -417,6 +417,7 @@ def marker(self):
         Marker can be set as follows::
 
             {'type': 'circle', 'color': 'cyan', 'radius': 20}
+            {'type': 'point', 'color': 'green', 'radius': 20, 'style': 'plus'}
```

Comment:
I'd like to make this `'type': 'plus'` and make a separate marker whose type is `'cross'` for the other ginga point option. 

What you have done is perfectly reasonable in that it follows how ginga defines the `Point` shape; since the goal in this package is to eventually provide a few different backends, though, I'd like to keep the point properties as uniform as possible and the names as explicit as possible.